sos/report/plugins/aap_controller.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2023 Pavel Moravec <pmoravec@redhat.com>
+# Copyright (c) 2024 Lucas Benedito <lbenedit@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class AAPControllerPlugin(Plugin, RedHatPlugin):
+    short_desc = 'AAP Automation Controller plugin'
+    plugin_name = 'aap_controller'
+    profiles = ('sysmgmt', 'ansible')
+
+    packages = ('automation-controller-venv-tower',
+                'automation-controller-server',
+                'automation-controller-ui',
+                'automation-controller')
+    commands = ('awx-manage')
+    services = ('automation-controller')
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/tower/",
+            "/etc/receptor/",
+            "/etc/supervisord.conf",
+            "/etc/supervisord.d/*",
+            "/var/log/tower",
+            "/var/log/nginx/automationcontroller.access.log*",
+            "/var/log/nginx/automationcontroller.error.log*",
+            "/var/log/supervisor",
+            "/var/log/unattended-upgrades",
+        ])
+
+        self.add_forbidden_path([
+            "/etc/tower/SECRET_KEY",
+            "/etc/tower/tower.key",
+            "/etc/tower/awx.key",
+            "/etc/tower/tower.cert",
+            "/etc/tower/awx.cert",
```

Comment:
Should these not be expanded to something like `/etc/tower/*.key` and `/etc/tower/*.cert`? I don't know if it's possible for other filenames to be used here, but if so I imagine we don't want to copy any of them.

Reply:
I *think* - in general - that all keys (private part of cert) should be forbidden, while there can be some reason to copy a cert file (public part, yet internally for the deployment) in some cases. Don't know how this applies to the specific keys / certs in `/etc/tower`.

Reply:
Changed it as suggested.